recipes/emsdk/all/conanfile.py

```diff
@@ -176,16 +176,16 @@ def package_info(self):
             os.path.join("bin", "upstream", "lib", "cmake", "llvm"),
         ]
 
-        # TODO: conan v1 stuff, to remove in conan v2?
-        self.env_info.PATH.extend(self._paths)
-        self.env_info.CONAN_CMAKE_TOOLCHAIN_FILE = toolchain
-        self.env_info.EMSDK = self._emsdk
-        self.env_info.EMSCRIPTEN = self._emscripten
-        self.env_info.EM_CONFIG = self._em_config
-        self.env_info.EM_CACHE = self._em_cache
-        self.env_info.CC = self._define_tool_var("emcc")
-        self.env_info.CXX = self._define_tool_var("em++")
-        self.env_info.AR = self._define_tool_var("emar")
-        self.env_info.NM = self._define_tool_var("emnm")
-        self.env_info.RANLIB = self._define_tool_var("emranlib")
-        self.env_info.STRIP = self._define_tool_var("emstrip")
+        if Version(conan_version).major < 2:
+            self.env_info.PATH.extend(self._paths)
+            self.env_info.CONAN_CMAKE_TOOLCHAIN_FILE = toolchain
+            self.env_info.EMSDK = self._emsdk
+            self.env_info.EMSCRIPTEN = self._emscripten
+            self.env_info.EM_CONFIG = self._em_config
+            self.env_info.EM_CACHE = self._em_cache
+            self.env_info.CC = self._define_tool_var("emcc")
```

Comment:
would this not be useful to have with build env?

Reply:
found it https://github.com/conan-io/conan-center-index/pull/13600/files#diff-0f83176c5b3c7cd12cc71869c6204c8c2d6600e3f97d518f984edbe12d63e928R163